bin/start

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 Intel Corporation
```

Comment:
Waiting for official guidance on copyright notices.

Reply:
I don't think the final decision will be 2017, Intel. Rather than copy-pasting while awaiting an official decision, we should adopt something to reflect the need for an answer:
Copyright TODO
Copyright 2018 The Sawtooth NEXT Directory Authors
Copyright 2018 TBD